AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -343,6 +343,7 @@ livejournal.com,tecmundo.com.br,rap.ru,abcnyheter.no##.fc-ab-root
 !
 @@||ebay-kleinanzeigen.de/static/js/belen/tracking/adblock-detection/prebid*.js
 @@||ebay-kleinanzeigen.de/static/js/belen/tracking/adblock-detection/advertising*.js
+alpinforum.com##.page-body > div[id][style*="position: fixed"][style*="z-index: 1;"]
```

Comment:
Why not `@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=alpinforum.com`?
@BlazDT please take a look.

Reply:
Why we should allow ad script if we can just hide the adblock message?

Reply:
I agree, it can be just hidden.